<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DbExportWindow</class>
 <widget class="QDialog" name="DbExportWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>361</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>按语言导出数据库</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
     <property name="topMargin">
      <number>20</number>
     </property>
     <item>
      <widget class="QLineEdit" name="epEd1"/>
     </item>
     <item>
      <widget class="QPushButton" name="epBn1">
       <property name="text">
        <string>文件...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,1">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QComboBox" name="epCb2"/>
     </item>
     <item>
      <widget class="QPushButton" name="epBn2">
       <property name="text">
        <string>导出...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="epPte3"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
